render.with_scaling.yaml

```
@@ -46,18 +46,16 @@ services:
       targetMemoryPercent: 80
       targetCPUPercent: 80
     envVars:
-      - key: FRONTEND_GRPC_PORT
-        value: 10000
       - key: SERVICES
         value: frontend
       - key: DYNAMIC_CONFIG_FILE_PATH
         # value: /etc/temporal/dynamicconfig.yaml
         value: /etc/temporal/dynamicconfig_es.yaml # Use this if deploying with ElasticSearch
-      - key: TEMPORAL_CLI_ADDRESS
+      - key: RENDER_FRONTEND_HOST
```

We tend to reserve `RENDER_*` for env vars that Render automatically sets in the environment: https://render.com/docs/environment-variables
```suggestion
      - key: TEMPORAL_FRONTEND_HOST
```